newrelic-agent/src/main/java/com/newrelic/agent/Agent.java

```diff
@@ -54,8 +49,7 @@
 import java.util.logging.Level;
 import java.util.regex.Pattern;
 
-import static com.newrelic.agent.config.SecurityAgentConfig.addSecurityAgentConfigSupportabilityMetrics;
-import static com.newrelic.agent.config.SecurityAgentConfig.shouldInitializeSecurityAgent;
+import static com.newrelic.agent.config.SecurityAgentConfig.*;
```

Comment:
We don't use * imports. Should be refactored to individual imports.

Reply:
Updated